Wattpad2epub.py

```diff
@@ -86,7 +87,23 @@
 print("saved "+ story_name+".html")
 print("Generating Epub...")
 
+cover_image = story_name+".jpg"
+urllib.request.urlretrieve(cover, cover_image)
+
+xmlfile = open('details.xml', 'r')
```

Comment:
Now, I'm not really sure if we should add XML file, I wanted to keep this simple. Since you wanted to add meta details for HTML, you don't need these steps.
It seems when the input is HTML, pandoc takes any available metadata settings from the HTML meta tags. So you can use meta tags in the head section directly in HTML.
I have already implemented it in my local repo, and it works, You can go ahead implement it if you want, ask anything if you need help.

Reply:
I had downloaded a book using your code and opened the Epub file in my phone on an Epub reader app (Lithium). The name of the book had been "Copied_73798392..." and the author had been "N/A" and there had been no cover image as well. I ran it after creating the XML file. The correct name, author and book was present. So I don't think the HTML tags were taken by pandoc. Or atleast it didn't work in that app.

![lithium](https://user-images.githubusercontent.com/42782061/121078055-126dcc80-c7f6-11eb-84db-5d783fa4d9f6.jpeg)

Reply:
Think I wasn't clear before:
```
file.write(f"<html><head>\
	<meta name='title' content='{story_name}'>\
	<meta name='author' content='{author}'>\
	</head><body>")
```
Try adding this in head section of HTML, instead of all XML code, I don't know how it would show across different EPUB apps, but according to pandoc this is valid metadata for epub and it does show the same metadata previously implemented.

Now for the cover-image, we haven't really changed the code for the cover (just a different library for downloading) image from before so it should still show up, try tinkering with it, if it still doesn't show up.

Reply:
I checked it on Lithium, it works.